actions/new.action.ts

```diff
@@ -674,33 +654,63 @@ const copyEnvFile = async (dir: string, envExample: string, envFile: string) =>
 };
 
 const createRegistry = async (
-  dir: string,
-  shouldInitializePrisma: boolean,
-  shouldInitializeUserService: boolean,
-  shouldInstallMonitoring: boolean,
-  shouldInitializeMonitoring: boolean,
-  shouldInitializeTemporal: boolean,
-  shouldInitializeLogging: boolean,
-  shouldInitializeFileUpload: boolean
+  projectDirectory: string,
+  spec: {
+    projectName: string,
+    prismaSetup: boolean,
+    userServiceSetup: boolean,
+    monitoringSetup: boolean,
+    monitoringServicesSetup: boolean,
+    temporalSetup: boolean,
+    loggingSetup: boolean,
+    fileUploadSetup: boolean,
+    packageManager: string
+  }
 ): Promise<void> => {
-  const filePath = join(process.cwd(), dir, '.stencil');
-  
-  const setupInfo = [
-    shouldInitializePrisma ? 'Prisma Setup' : '',
-    shouldInitializeUserService ? 'User Services Setup' : '',
-    shouldInstallMonitoring ? 'Monitoring Installed' : '',
-    shouldInitializeMonitoring ? 'Monitoring Setup' : '',
-    shouldInitializeTemporal ? 'Temporal Setup' : '',
-    shouldInitializeLogging ? 'Logging Setup' : '',
-    shouldInitializeFileUpload ? 'File Upload Setup' : ''
-  ].filter(info => info !== '').join('\n');
+  const { projectName, packageManager, ...setupInfo } = spec;
 
+  const filePath = join(process.cwd(), projectDirectory, 'spec.yaml');
```

Comment:
Why is the file path hardcoded?

Reply:
The spec.yaml file will be generated everytime inside the root folder of the project right?
if `stencil spec temp/spec.yaml` is being run by the user and the spec.yaml has projectName `final` then the spec.yaml will be generated inside of root of `final` project folder, right?